shell/src/main/java/org/apache/accumulo/shell/commands/MergeCommand.java

```diff
@@ -95,11 +95,9 @@ public Options getOptions() {
         new Option("s", "size", true, "merge tablets to the given size over the entire table");
     forceOpt = new Option("f", "force", false,
         "merge small tablets to large tablets, even if it goes over the given size");
-    // Using the constructor does not allow for empty option
-    Option.Builder builder = Option.builder().longOpt("all").hasArg(false)
```

Comment:
If want to avoid adding the `-a` option then maybe could keep this builder code as is and change the following code which is a bit further up in the class from

```java
    if (cl.hasOption(allOpt.getOpt())) {
      all = true;
    }
```

to

```java
    if (cl.hasOption(allOpt.getLongOpt())) {
      all = true;
    }
```

Reply:
Man, should have thought of that....

Reply:
I personally don't see an issue with adding a `-a` short option but I understand wanting to keep the code as minimal as possible.

Reply:
I put up #5456 to just check the option and not add the '-a' short option.

Reply:
> Do we really need to decide which option string to check for?
CommandLine has a hasOption method that takes the entire Option object instead of choosing between the short and long opt values.
https://commons.apache.org/proper/commons-cli/apidocs/org/apache/commons/cli/CommandLine.html#hasOption(Option)


I wonder if all the code should use this and if it means some of the existing code is not handling long options properly.  Like will only the `-f` options work for the merge command and not the `--force` option?

Reply:
The `getOptionValue` has a similar issue. If you use `getOptionValue(Option)`, then you should be good. If you use `getOptionValue(String)`, then you might be missing something.

Reply:
> I wonder if all the code should use this and if it means some of the existing code is not handling long options properly. Like will only the -f options work for the merge command and not the --force option?

Looked into this and it seems to work.  When `-f` string is passed to CLI code it will use that to find the Option object and would still match if `--force` was passed in, but that only works when there is a short option.  Still seems better to just pass the option object though.